tests/meta/conftest.py

```diff
@@ -167,67 +154,106 @@ def __init__(self):
         super().__init__(config={"microsalt": {"root": tmpdir}})
 
 
-def safe_loader(path):
-    """Mock function for loading yaml"""
-
-    if path:
-        pass
+@pytest.yield_fixture(scope="function")
+def mip_hk_store(helpers, real_housekeeper_api, timestamp):
+    deliver_hk_bundle_data = {
+        "name": "case_id",
+        "created": timestamp,
+        "expires": timestamp,
+        "files": [
+            {
+                "path": "tests/fixtures/apps/mip/dna/store/case_config.yaml",
+                "archive": False,
+                "tags": ["mip-config"],
+            },
+            {
+                "path": "tests/fixtures/apps/mip/dna/store/case_qc_sample_info.yaml",
+                "archive": False,
+                "tags": ["sampleinfo"],
+            },
+            {
+                "path": "tests/fixtures/apps/mip/case_qc_metrics.yaml",
+                "archive": False,
+                "tags": ["qcmetrics"],
+            },
+            {
+                "path": "tests/fixtures/apps/mip/case_file.txt",
```

Comment:
if this is a "real" mip-file, is it called like that? otherwise I think we should call it what mip calles it, ditto next one

Reply:
its a dummy file to test case tag